gpsd/__init__.py

```diff
@@ -128,6 +138,10 @@ def from_json(cls, packet):
             result.error['c'] = last_tpv['epc'] if 'epc' in last_tpv else 0
             result.error['v'] = last_tpv['epv'] if 'epv' in last_tpv else 0
 
+        result.heading = last_att['heading'] if 'heading' in last_att else 0.0
```

Comment:
What happens if the actual value is 0.0?

Reply:
What do you mean by _actual value_? Do you mean `result.heading`?

Reply:
If you take a look at the heading and it is 0.0 you assume north I guess. So i think you can't use 0.0 to tell the user that there is no heading information. I think it should be something like None or so.

Reply:
Okay.  Now I understand.  Yes, 0 is a valid heading, so the error condition should return `None`. I’ll make the change and push.